handbook/superset.md

```diff
@@ -0,0 +1,88 @@
+# Superset
+
+## What is Superset
+Superset is a way to query, visualize, and explore the data. Currently, the data hosted on Superset uses Postgres as the meta database engine.
+
+## Accessing Superset in Web Browser
+In order to access Superset, you would first need access to FASSE and the NSAPH host (instructions [here](fasse.md))
+
+Open Terminal, and enter the following command: 
+```
+ssh -L8088:localhost:8088 username@nsaph.rc.fas.harvard.edu
+```
+After logging in through this specific port, you can access the Superset interface locally. In your browser, enter the following url: `http://localhost:8088/login/`
+
+Enter your Superset credentials.
+```{warning}
+If you don't know your Superset credentials, stop immediately and contact us.
+```
+
+Once the Superset authentication is completed, you can now explore the datasets you are granted access to. Start by going to SQL Editor under SQL Lab.
+
+```{figure} imgs/superset_welcome.png
+---
+scale: 30%
+align: center 
+---
+Accessing SQL Lab.
+```
+
+## Access Superset with Jupyter Lab
+Accessing Superset via Jupyter Lab will allow you to query the databases and create data query pipelines with Python. The environment is already configured to be accessible to all NSAPH users. 
+
+```{note}
+If you would like to customize your own environment, follow this [section](customize). Not sure if you need to? Do NOT customize it.
+```
+
+Step 1: Login to the NSAPH host
+```
+ssh -L8088:localhost:8088 username@nsaph.rc.fas.harvard.edu
+```
+
+Step 2: Initialize the Anaconda environment
+```
+/opt/anaconda3/condabin/conda init
+```
+```{note}
+You may be prompted to restart the host. If that is the case, exit the NSAPH host and login again. You will only need to do step 2 once. All subsequent logins to NSAPH host will activate the Anaconda automatically.
+```
+Step 3: Check the available environments
+```
+conda env list
+```
+You should find an environment called `superset_final`. Activate that environment.
```

Comment:
Are you sure that all users will see this environment?

Reply:
Maybe you could export the conda environment in the `environment.yml` file and add it here (or link to it).

This is how you create the .yml file: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#exporting-the-environment-yml-file
(just run `conda env export > environment.yml` from the environment)